### DISCUSSION TITLE
Enable Bun

### DISCUSSION THREAD

**Discussion by AllanOricil** [forcedotcom/cli#2457] [Ideas]
Bun 1.0 has finally been launched!
https://bun.sh/blog/bun-v1.0

obs: windows runtime is still experimental

"While the macOS and Linux builds of Bun are production-ready, the Windows build is highly experimental"

But if your telemetry data show that most users use mac/Linux, then why not? I would give it a try

**Comment by mshanemc** [comment]
windows is the most common OS.

**Reply by AllanOricil** [reply]
Fuck 😅

**Comment by AllanOricil** [comment]
Well, I would still create a benchmark to see how much faster the cli is when using bun. Then share this benchmark with plugin creators. They can just run it and post the results in their README to convince people to switch. Maybe this can be an Ocliff feature to spread the word faster.

**Reply by mshanemc** [reply]
yeah, this would definitely be an oclif feature.
